src/core/hooks/createDataAttribute/createDataAttribute.test.tsx

```
@@ -0,0 +1,54 @@
+  const { result: accentAttributes } = renderHook(() =>
+    useCreateDataAttribute("accent", { color: "" })
+  );
+
+  // Merge the attributes
+  const { result: composedAttributes } = renderHook(() =>
+    useComposeAttributes(dataAttributes.current(), accentAttributes.current())
+  );
+
+  // Assert that only the valid `data-*` attributes are present
+  expect(composedAttributes.current()).toEqual({
+    "data-button-size": "large", // "variant" is ignored since it's undefined, and "color" is ignored since it's an empty string
```

ESLint

[error] Unexpected trailing comma.

(comma-dangle)